src/hub.js

```diff
@@ -209,6 +215,7 @@ const onReady = async () => {
 
   const enterScene = async (mediaStream, enterInVR, janusRoomId) => {
     const scene = document.querySelector("a-scene");
+    scene.renderer.sortObjects = true;
```

Comment:
does this need to be reverted in `exitScene`?

Reply:
I don't think so. I just need to make sure `sortObjects` is enabled -- I stuck it here because the scene must exist and have a renderer by the time we're entering it.

Reply:
I know we were already doing this, but is there a cost to turning this on and does it increase only with the cost of non-zero renderOrder objects or just total number of objects in the scene?

Reply:
It scales with the total number of objects in the render list. See 
https://github.com/mrdoob/three.js/blob/master/src/renderers/WebGLRenderer.js#L1123
and
https://github.com/mrdoob/three.js/blob/36444d5660685cf153b99e49c126b5043e4edefd/src/renderers/webgl/WebGLRenderLists.js#L5